profiles/license_groups

```diff
@@ -30,7 +30,7 @@ OSI-APPROVED AFL-3.0 AGPL-3 AGPL-3+ Apache-1.1 Apache-2.0 APL-1.0 APSL-2 Artisti
 # Licenses in this list should NOT appear directly or indirectly in
 # @FSF-APPROVED or @OSI-APPROVED.
 # Note: Licenses for fonts should be included in @MISC-FREE-DOCS.
-MISC-FREE AIFFWriter.m Allegro alternate AMPAS bea.ri.jsr173 BEER-WARE boehm-gc BSD-1 BSD-with-attribution buddy bufexplorer.vim BZIP2 canfep CAOSL CDDL-1.1 CDDL-Schily CeCILL-C CMake CPL-0.5 CRACKLIB Crypt-IDEA DES docbook dom4j DUMB-0.9.3 eGenixPublic-1.1 ElementTree Emacs ErlPL-1.1 FastCGI feh File-MMagic Flashpix FLEX flexmock FLTK freetts FVWM gd gsm HTML-Tidy htmlc iASL icu IDPL imagemagick Info-ZIP inner-net Interbase-1.0 ipadic ipx-utils Ispell JasPer2.0 JDOM JNIC JOVE Khronos-CLHPP LambdaMOO LIBGLOSS libmng libpng libpng2 libtiff LLVM-Grant LPPL-1.3 LPPL-1.3b lsof Mail-Sendmail mapm-4.9.5 matplotlib Mini-XML minpack MIT-with-advertising mm mpich2 NCSA-HDF netcat NEWLIB ngrep Old-MIT openafs-krb5-a Openwall otter PCRE perforce photopc PHP-2.02 pngcrush pngnq Princeton psutils qmail-nelson rc rdisc regexp-UofT repoze RSA rwpng scanlogd Sendmail Sendmail-Open-Source shrimp SMAIL Snd SNIA SSLeay Subversion SVFL symlinks tablelist tcltk tcp_wrappers_license TeX TeX-other-free the-Click-license Time-Format Time-modules tm-align torque-2.5 totd Toyoda UCAR-Unidata URT VTK w3m x2x xbatt xboing XC Xdebug xtrs xvt YaTeX ZSH
+MISC-FREE AIFFWriter.m Allegro alternate AMPAS bea.ri.jsr173 BEER-WARE boehm-gc BSD-1 BSD-with-attribution buddy bufexplorer.vim BZIP2 canfep CAOSL CDDL-1.1 CDDL-Schily CeCILL-C CMake CPL-0.5 CRACKLIB Crypt-IDEA DES docbook dom4j DUMB-0.9.3 eGenixPublic-1.1 ElementTree Emacs ErlPL-1.1 FastCGI feh File-MMagic Flashpix FLEX flexmock FLTK freetts FVWM gd gsm HTML-Tidy htmlc iASL icu IDPL imagemagick Info-ZIP inner-net Interbase-1.0 ipadic ipx-utils Ispell JasPer2.0 JDOM JNIC JOVE Khronos-CLHPP LambdaMOO LIBGLOSS libmng libpng libpng2 libtiff LLVM-Grant LPPL-1.3 LPPL-1.3b lsof Mail-Sendmail mapm-4.9.5 matplotlib meschach Mini-XML minpack MIT-with-advertising mm mpich2 NCSA-HDF netcat NEWLIB ngrep Old-MIT openafs-krb5-a Openwall otter PCRE perforce photopc PHP-2.02 pngcrush pngnq Princeton psutils qmail-nelson rc rdisc regexp-UofT repoze RSA rwpng scanlogd Sendmail Sendmail-Open-Source shrimp SMAIL Snd SNIA SSLeay Subversion SVFL symlinks tablelist tcltk tcp_wrappers_license TeX TeX-other-free the-Click-license Time-Format Time-modules tm-align torque-2.5 totd Toyoda UCAR-Unidata URT VTK w3m x2x xbatt xboing XC Xdebug xtrs xvt YaTeX ZSH
```

Comment:
Nope, this is not a free software license. The problem is the "no charge" clause.

Generally, it is strongly recommended to file a bug for the licenses team before adding a new license to one of the groups, unless it is an obvious case (e.g, a license approved by FSF or OSI).

Reply:
It's DFSG and the question was already raised on debian-legal ML

https://lists.debian.org/debian-legal/2000/03/msg00025.html

Reply:
> Everyone is granted permission to copy, modify and redistribute this Meschach Library, provided: [...]
> 3.  No charge is made for this software or works derived from it. [...]

The [Open Source Definition](https://opensource.org/osd) says:
> **1. Free Redistribution**
> The license shall not restrict any party from selling or giving away the software as a component of an aggregate software distribution containing programs from several different sources. The license shall not require a royalty or other fee for such sale.

Seems pretty clear that this isn't a free license. IMHO it doesn't even qualify for our `BINARY-REDISTRIBUTABLE` group.